Lab1_Operationalizing_Pytorch_with_Mlflow/src/app/app.py

```diff
@@ -0,0 +1,31 @@
+from flask import Flask, request, render_template
+import requests
+
+app = Flask(__name__)
+app.config.from_object("settings")
+
+
+@app.route('/')
+def home():
+    return render_template('index.html')
+
+
+@app.route('/predict', methods=['POST'])
+def predict():
+    model_serve_url = app.config["SERVE_URI"]
+    if model_serve_url is None:
+        return render_template('index.html', prediction_text="ERROR: Model serve url not set")
+
+    news_text = request.form["news"]
+
+    resp = requests.post(f"{model_serve_url}/predictions/news_classification", data=news_text)
+    if resp.ok:
+        output = f"The article is of type {resp.text}"
+    else:
+        output = f"Error in model response:\n{resp.text}"
+
+    return render_template('index.html', prediction_text=output, input=news_text)
+
+
+if __name__ == "__main__":
+    app.run(debug=True)
```

Comment:
Wouldn't be better _FLASK_ENV=development_ (or something like that) instead of _debug=True_?

Reply:
Good catch, removed this here and added description to README